backend/src/index.ts

```diff
@@ -11,12 +11,12 @@ connect(config.MONGODB_URI)
   .catch((err) => logger.error('error connecting to mongodb:', err.message));
 
 const app = express();
-const port = 3000;
+const port = 5000;
```

Comment:
I forgot to do this but we can probably define this in .env and reference it from config.js

Reply:
usually it's 3000 for FE and 3001 for BE but doesn't really matter once we have it as .env

Reply:
True I can swap the values out now